common/src/main/java/org/apache/celeborn/common/util/PbSerDeUtils.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.celeborn.common.meta.FileInfo;
 import org.apache.celeborn.common.meta.WorkerInfo;
 import org.apache.celeborn.common.protocol.*;
+import org.apache.celeborn.common.protocol.PartitionLocation.Mode;
 import org.apache.celeborn.common.protocol.message.ControlMessages.*;
 
 public class PbSerDeUtils {
```

Comment:
I see. I check the Utils and seems it have only one couple of conversion which is `convertPbWorkerResourceToWorkerResource`/`convertWorkerResourceToPbWorkerResource`, and we have migrated and added new conversions such as `fromPbDiskInfo`/`toPbDiskInfo` `fromPbUserResourceConsumption`/`toPbUserResourceConsumption` which are not used in worker recovery in PbSerDeUtils before, so maybe we can extract them into PbSerDeUtils?
BTW, do we need to transform PbSerDeUtils to scala code, seems it will be convenient to handle the transformation.
cc: @AngersZhuuuu @waitinfuture